### PR TITLE
Resetting the pipeline application list after test

### PR DESCRIPTION
**Pull request by sn-px**
**What this PR does / why we need it**:
Resetting the pipeline application list to make the execution of the other testcases to run smooth

**Special notes for your reviewer**:

### CODE REVIEW

File: tests/backup/backup_psa_test.go [hunk 1]
```diff
@@ -898,6 +899,10 @@ var _ = Describe("{RestoreFromHigherPrivilegedNamespaceToLower}", Label(TestCase
 	})
 	JustAfterEach(func() {
 		defer EndPxBackupTorpedoTest(scheduledAppContexts)
+
+		//Resetting the pipeline app list
+		Inst().AppList = pipelineAppList
```
Comment (kshithijiyer-px) [inline tests/backup/backup_psa_test.go]: Let's put it in defer so that it doesn't break the next testcases:
```
		defer func() {

		}()
```

Comment (sn-px) [reply tests/backup/backup_psa_test.go]: done

**Review by kshithijiyer-px — APPROVED** [review]
LGTM

File: tests/backup/backup_psa_test.go [hunk 2]
```diff
@@ -898,6 +899,12 @@ var _ = Describe("{RestoreFromHigherPrivilegedNamespaceToLower}", Label(TestCase
 	})
 	JustAfterEach(func() {
 		defer EndPxBackupTorpedoTest(scheduledAppContexts)
+
+		//Resetting the pipeline app list
+		defer func() {
+			Inst().AppList = pipelineAppList
+		}()
+
```
Comment (mkoppal-px) [inline tests/backup/backup_psa_test.go:907]: The defer should come in `JustBeforeEach` itself. Refer to other tests where we have done this.

Comment (sn-px) [reply tests/backup/backup_psa_test.go]: done

**Review by mkoppal-px — APPROVED** [review]
LGTM

**Pull request merged**